### ISSUE TITLE
Extraction Pump doesn't work

### ISSUE THREAD

**Issue by Sicrosis** [TheLimePixel/Modern-Skyblock-3#801]
Will not pull liquids in any form and as such road blocks progression. Make those machines bucketable.

**Comment by StephenLujan** [comment]
I experienced a similar issue, but there's a work around. The Extraction pump only seems to work for me only when placed with the correct orientation, not when rotated with the monkey wrench to the correct orientation. Because shift-right-click causes you to place it facing away from you, but is necessary to avoid opening some machine interfaces, you may want to use a placeholder block to add the extraction pump, then replace the placeholder with the machine or tank after, or shift-right-click on the pipe it will output to. 

Please let me know if you have the same experience.

**Comment by StephenLujan** [comment]
So I have reported this issue to Factory Tech as well as I felt it unlikely the issue would be specific to this mod pack.
https://gitlab.com/dalapo/FactoryTech/issues/41

**Comment by Sicrosis** [comment]
Yup, when placing without having to wrech it will work, sadly that means breaking the machine which breaks all parts inside. Kind of a pain. but at least it works now, Thanks

**Comment by StephenLujan** [comment]
Actually I was also having trouble automating part input, and I thought I just didn't understand something, but I'm starting to wonder if the part input side was not actually changing with the monkey wrench despite appearing to with the magnifying glass.

**Comment by StephenLujan** [comment]
> Yup, when placing without having to wrech it will work, sadly that means breaking the machine which breaks all parts inside. Kind of a pain. but at least it works now, Thanks

Remember you can also shift-right-click the pipe/tank/machine you want to output to, rather than the input, and it should work.

**Comment by StephenLujan** [comment]
Fixed in version 1.6b of Factory Tech, along with quite a few other problems since 1.5.5 as in this pack. If you're not in a public server you can probably safely update this mod yourself until EmosewaPixel has time to make an update.

**Comment by DrunkTaank** [comment]
> I experienced a similar issue, but there's a work around. The Extraction pump only seems to work for me only when placed with the correct orientation, not when rotated with the monkey wrench to the correct orientation. Because shift-right-click causes you to place it facing away from you, but is necessary to avoid opening some machine interfaces, you may want to use a placeholder block to add the extraction pump, then replace the placeholder with the machine or tank after, or shift-right-click on the pipe it will output to.
> 
> Please let me know if you have the same experience.

Thank you for posting this.  It was driving me nuts wondering why the pump wouldn't work
